components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -826,10 +826,12 @@ function BasePrizePool:storeData()
 	end
 
 	for _, lpdbEntry in ipairs(lpdbData) do
-		lpdbEntry.lastvsdata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.lastvsdata or {})
-		lpdbEntry.opponentplayers = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.opponentplayers or {})
-		lpdbEntry.players = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.players or {})
-		lpdbEntry.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.extradata or {})
+		Table.mergeInto(lpdbEntry, {
+			lastvsdata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.lastvsdata or {}),
+			opponentplayers = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.opponentplayers or {}),
+			players = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.players or {}),
+			extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.extradata or {}),
+		})
```

Comment:
Can Json.stringifySubTables be used instead?

Reply:
yes, would need testing, so probably monday or tuesday